### PR TITLE
package: update strong-supervisor to 5.x

### PR DESCRIPTION
**Pull request by sam-github**
5.x uses appmetrics instead of strong-agent

connected to https://github.com/strongloop-internal/scrum-nodeops/issues/1465

### CODE REVIEW

**Comment by sam-github** [comment]
@rmg PTAL - fails windows, but I believe that to be pre-existing, Windows will require another painful sweep through CI later

**Comment by rmg** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
```

> appmetrics@1.1.0 install /Users/jenkins/build/workspace/strong-runner/d2063294/node_modules/strong-supervisor/node_modules/appmetrics
> node extract_all_binaries.js

Thu, 06 Oct 2016 22:49:39 GMT
********************************************************************************
You are installing the Node Application Metrics monitoring and profiling module.
Licensed under the Apache License, Version 2.0 (the "License")
...
********************************************************************************
Download and extract of 0.10/appmetrics-1.1.0-darwin-x64.tgz finished.
Download and extract of agentcore-3.0.9-darwin-x64.tgz finished.
SAM -------- weird, extract finished, so what tarball is 0-byte?
npm ERR! Darwin 15.6.0
npm ERR! argv "node" "/opt/node-0.10/bin/npm" "install"
npm ERR! node v0.10.45
npm ERR! npm  v2.15.6

npm ERR! 0-byte tarball
npm ERR! Please run `npm cache clean`
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jenkins/build/workspace/strong-runner/d2063294/npm-debug.log
```

Its not appmetrics, it looks like something else went wrong

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
Passing on anything other than windows with node 5 and 6, good enough for now.

@rmg PTAL

File: lib/runner.js [hunk 1]
```diff
@@ -287,12 +288,23 @@ Runner.prototype._restart = function _restart(status) {
 
   // The runner should not exit unless we stopped it so a next version of the
   // app could be run, something has gone wrong!
+  // XXX(sam) though a fast stop followed by a start can confuse this heuristic,
+  // to make it more accurate we would need a better state machine to track the
+  // current state of the app, and its not so clear the criteria for liveness.
+  // In practice, its only wrong in unit tests, and the message is only a
```
Comment (rmg) [inline lib/runner.js]: `s/its/it's/`

**Comment by rmg** [comment]
The windows failures are a bit concerning since they look pretty fundamental, but if you're ok with them then I guess they aren't a problem.

**Comment by sam-github** [comment]
Its weird, its passing on < 4, so it looks like something changed with how links are handled in >= 4. But the main thing is, whether the code is actually broken now, or just the tests, is that its completely unrelated to the appmetrics update.

**Comment by sam-github** [comment]
Fixed the typos, thanks.

**Pull request merged**